src/components/UnlockWallet/index.tsx

```diff
@@ -30,6 +31,49 @@ function warnUser(title: string, message?: string | undefined): void {
     message,
   });
 }
+function changeId(chainName : any) {
+  var chains =  JSON.parse(JSON.stringify(internalAddressesPerNetwork))
```

Comment:
This line is an expensive no-op. Also, use `const`.

Reply:
Removed